src/seeders/01-demo-user.js

```diff
@@ -43,8 +48,10 @@ module.exports = {
       firstName: 'John',
       lastName: 'Doe',
       email: 'demo4@demo.com',
-      username: 'user2',
+      username: 'user4',
       password: await init(),
+      isVerified: true,
+      role: 'Manager',
```

Comment:
Since we will be giving the user a default value there is no need to set a role here. you don;t need to change it. am just saying.